cmd/lambda/usage/get.go

```diff
@@ -86,38 +92,13 @@ func GetUsageByStartDateAndPrincipalID(w http.ResponseWriter, r *http.Request) {
 		usageResponseItems = append(usageResponseItems, &usageRes)
 	}
 
-	json.NewEncoder(w).Encode(usageResponseItems)
-}
-
-// GetAllUsage - Returns a list of entire usage
-func GetAllUsage(w http.ResponseWriter, r *http.Request) {
```

Comment:
Method isn't used anywhere

@AmudaPalani are we missing an integration with our router here?